### ISSUE TITLE
Initialize error

### ISSUE THREAD

**Issue by uga-rosa** [b4b4r07/afx#36]
## WHAT

If I take my currently functioning `~/.config/afx/*` under the new environment, put the latest release of the `afx` binary in the path, and run `afx install`, I get an error that `~/.afx/state.json` does not exist.

**Comment by babarot** [comment]
Thank you for using this tool and reporting the trouble. 

I'm also using `~/.config/afx` as config directory but I didn't face this kinda situation in my env. So please let me know more detail log? [This page](https://babarot.me/afx/faq/#debugging) is helpful to show debug log.

**Comment by uga-rosa** [comment]
```
I9-1% AFX_LOG=debug afx install
2022/03/21 11:59:38 [INFO] afx version: 0.1.18
2022/03/21 11:59:38 [INFO] Go runtime version: go1.16.1
2022/03/21 11:59:38 [INFO] Build tag/SHA: v0.1.18/adb5a3e
2022/03/21 11:59:38 [INFO] CLI args: []string{"afx", "install"}
2022/03/21 11:59:38 [DEBUG] (goroutine): checking new updates...
2022/03/21 11:59:38 [INFO] Reading config /home/nakai/.config/afx/github.yaml...
2022/03/21 11:59:38 [DEBUG] call GitHub Release API to get release info
2022/03/21 11:59:38 [INFO] Parsing config...
2022/03/21 11:59:38 [INFO] Reading config /home/nakai/.config/afx/main.yaml...
2022/03/21 11:59:38 [INFO] Parsing config...
2022/03/21 11:59:38 [INFO] Reading config /home/nakai/.config/afx/release.yaml...
2022/03/21 11:59:38 [INFO] Parsing config...
2022/03/21 11:59:38 [DEBUG] dependency graph is here:
* sindresorhus/pure -> mafredri/zsh-async
2022/03/21 11:59:38 [DEBUG] mkdir /home/nakai/.afx
2022/03/21 11:59:38 [DEBUG] mkdir /home/nakai/.afx/bin
[ERROR]: failed to initialize afx: faield to open state file: open /home/nakai/.afx/state.json: no such file or directory
```

**Comment by babarot** [comment]
Thank you. This has been solved at new version [v0.1.19](https://github.com/b4b4r07/afx/releases/tag/v0.1.19)
